src/directory/generateDirectory.mjs

```diff
@@ -0,0 +1,97 @@
+import { promises as fs } from 'fs';
+import { fileURLToPath } from 'url';
+import { cwd } from 'node:process';
+import path from 'path';
+import JSON5 from 'json5';
+import { directory } from './directory.mjs';
+import { writeFile } from 'fs/promises';
+
+/**
+ * Helper function to use RegEx to grab the "meta" object
+ * @param {string} filePath
+ * @returns
+ */
+async function getMetaStringObj(filePath) {
+  const regex = /const\s+meta\s*=\s*(\{[^}]+\})/;
```

Comment:
Went with the regex approach to get the `meta` object. Was running into a lot of trouble just trying to compile the `mdx` file